### ISSUE TITLE
`colcon build package` should default to building package and dependencies

### ISSUE THREAD

**Issue by timonegk** [colcon/colcon-core#486]
With `catkin_tools`, it was simple to build a package with its dependencies, which is one of the most common use cases when building packages. With `colcon`, this can be done using the flag `--packages-up-to`, which is a bit cumbersome. I suggest to add `colcon build package` as a shorthand for `colcon build --packages-up-to`. Maybe also add `colcon build --no-deps package` for `colcon build --packages-select package`.

**Comment by cottsay** [comment]
Please take a look at the [colcon-alias](https://github.com/colcon/colcon-alias) extension. You should be able to add an alias to do something like what you're asking.

Setup: `colcon alias build-up-to build --packages-up-to`
Execution: `colcon build-up-to foo` would translate to `colcon build --packages-up-to foo`.

**Comment by cottsay** [comment]
Hi @timonegk, were you able to give `colcon-alias` a try?

**Comment by timonegk** [comment]
Yes, thanks, that works. (With `colcon alias --command build --packages-up-to -- build-up-to`.)
